ontology/yaml/resources/HVAC/entity_types/ABSTRACT.yaml

```diff
@@ -4061,6 +4107,8 @@ CLPM:
   is_abstract: true
   uses:
   - cooling_thermal_power_sensor
+  opt_uses:
+  - cooling_thermal_energy_accumulator
```

Comment:
Should this be an optional field or does it belong on its own abstract type? Review w Trevor

Reply:
Confirmed okay for now. Something we may want to revisit when we do a full dbo scrub